src/components/calculator/controls/VerticalControl.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useRef, useState } from 'react'
+import { ToggleButton, ToggleButtonGroup } from 'react-bootstrap'
+
+import { recordCalculatorOptionSelected } from 'components/Analytics'
+import { BaseFormValue } from 'data/data'
+
+export const VerticalControl: React.FunctionComponent<{
```

Comment:
Could this be more descriptive? I don't have a sense of how this is supposed to look based on the component name. Alternatively, add documentation?

Reply:
done